jest.config.mjs

```diff
@@ -65,6 +65,9 @@ export default {
     '/packages/jest-snapshot/src/__tests__/fixtures/',
     '/e2e/__tests__/iterator-to-null-test.ts',
     '/e2e/__tests__/tsIntegration.test.ts', // this test needs types to be build, it runs in a separate CI job through `jest.config.ts.mjs`
+    ...(process.env.JEST_JASMINE
+      ? ['/e2e/__tests__/testFailingSnapshot.test.js']
+      : []),
```

Comment:
please use https://github.com/jestjs/jest/blob/5aeee2efc5312f672ea16190ef37ecc45bc457fd/packages/test-utils/src/ConditionalTest.ts#L17 instead

Reply:
I initially had a similar approach but the CI checks failed because when it starts running the test file, it finishes with errors/warnings because of "unused" (skipped) snapshots that are used in the regular circus tests. I had to skip the file completely.

Reply:
Ahh gotcha. Sounds good